modules/animation/include/modules/animation/datastructures/basetrack.h

```diff
@@ -159,6 +163,35 @@ template <typename Seq>
 BaseTrack<Seq>::BaseTrack(const std::string& identifier, const std::string& name, size_t priority)
     : identifier_{identifier}, name_{name}, priority_{priority} {}
 
+template <typename Seq>
+BaseTrack<Seq>::BaseTrack(const BaseTrack<Seq>& other)
+    : KeyframeSequenceObserver(other)
+    , enabled_{other.enabled_}
+    , identifier_{other.identifier_}
```

Comment:
Good point. It would need to be unique within an animation. 
I have removed the identifier since it was basically not used and it would have been more complicated to enforce the uniqueness.